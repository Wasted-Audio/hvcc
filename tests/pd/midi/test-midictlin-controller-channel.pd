#N canvas 864 247 503 302 12;
#X obj 98 132 print;
#X obj 98 61 ctlin 13 3;
#X connect 1 0 0 0;
