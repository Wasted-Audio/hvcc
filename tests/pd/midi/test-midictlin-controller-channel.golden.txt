[@ 0.000] print: 59
[@ 0.000] print: 60
[@ 0.000] print: 61
[@ 0.000] print: 62
[@ 0.000] print: 63
[@ 0.000] print: 62
[@ 0.000] print: 63
[@ 0.000] print: 62
[@ 0.000] print: 61
[@ 0.000] print: 60
[@ 0.000] print: 59
[@ 0.000] print: 58